.gitignore

```diff
@@ -5,7 +5,6 @@
 
 # Github Actions master branch will not have node_modules (use v1 instead)
 node_modules/
-lib/
 
```

Comment:
What change has been done here ?

Reply:
Lib folder was mentioned in .gitignore. Because of which, all the .js files under lib were not getting committed that's why I removed the lib folder from here.